testapp/tests/test_models.py

```diff
@@ -0,0 +1,11 @@
+# -*- coding: utf-8 -*-
+from testapp.models import Entry, DBSession
+
+
+def test_create_entry(dbtransaction):
+    """Test for a change of state of the model."""
+    new_model = Entry(title="jill", text='jello')
+    assert new_model.id is None
+    DBSession.add(new_model)
+    DBSession.flush()
+    assert new_model.id is not None
```

Comment:
more things you might test about your models:

Can you create one with a title longer than 128 chars?
Can you create one with no title or text?
Does creating a new one automatically set the created datetime?